packages/client/hooks/useDraggableReflectionCard.tsx

```diff
@@ -59,8 +65,24 @@ const useRemotelyDraggedCard = (
         const newTimeRemaining = timeRemaining - (Date.now() - beforeFrame)
         setRemoteCard(isClose, newTimeRemaining, bbox.top)
       })
+    } else if (isSpotlight) {
+      // move animating remote Spotlight when other reflections move
+      spotlightAnimRef.current = requestAnimationFrame(() => {
+        const newTimeRemaining = timeRemaining - (Date.now() - beforeFrame)
+        setRemoteCard(isClose, newTimeRemaining, bbox.top, isSpotlight)
```

Comment:
Update the remote Spotlight reflection position so that when other cards in the kanban move, the animating Spotlight moves rather than hovers over another card

Reply:
To test:
- open the Spotlight in client A and see that the remote reflection is animating in a circle in client B
- drag a reflection on the kanban that's positioned above the animating Spotlight 
- see that the animating remote reflection adjusts as the kanban reflections change position. Previously, the animating Spotlight didn't change position, meaning it would hover above other reflections on the kanban

In the Loom, you saw this bug, but then couldn't reproduce it later on. This was because you were on master initially and then switched to this branch, so it should be all fixed now.